src/types/api.types.ts

```diff
@@ -0,0 +1,7 @@
+export interface LoadingState {
+  loading: boolean;
+}
+
+export interface ErrorState {
+  error: string | undefined | null;
+}
```

Comment:
Like how is described [here](https://github.com/monstar-lab-oss/admin-panel-template-reactjs/discussions/88#discussioncomment-942054
), it would be better to write like below

```suggestion
export type LoadingState {
  loading: boolean;
}

export type ErrorState {
  error: string | undefined | null;
}
```

Reply:
Thanks, done